2018-08-20 18:54:15 ERROR DatabaseDao:41 - No value specified for parameter 3.
2018-08-20 18:54:15 ERROR DatabaseDao:43 - org.postgresql.core.v3.SimpleParameterList.checkAllParametersSet(SimpleParameterList.java:257)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:290)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:132)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - com.revature.daos.DatabaseDao.createUser(DatabaseDao.java:37)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - com.revature.screens.AddNewClientScreen.start(AddNewClientScreen.java:46)
2018-08-20 18:54:15 ERROR DatabaseDao:43 - com.revature.launcher.Launcher.main(Launcher.java:10)
2018-08-20 18:54:15 WARN  DatabaseDao:45 - failed to create new user
2018-08-20 19:16:06 ERROR DatabaseDao:41 - No value specified for parameter 5.
2018-08-20 19:16:06 ERROR DatabaseDao:43 - org.postgresql.core.v3.SimpleParameterList.checkAllParametersSet(SimpleParameterList.java:257)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:290)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:132)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - com.revature.daos.DatabaseDao.createUser(DatabaseDao.java:37)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - com.revature.screens.AddNewClientScreen.start(AddNewClientScreen.java:46)
2018-08-20 19:16:06 ERROR DatabaseDao:43 - com.revature.launcher.Launcher.main(Launcher.java:10)
2018-08-20 19:16:06 WARN  DatabaseDao:45 - failed to create new user
2018-08-20 19:26:15 ERROR DatabaseDao:41 - ERROR: column "pass" of relation "pyramid_users" does not exist
  Position: 38
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2178)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:132)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - com.revature.daos.DatabaseDao.createUser(DatabaseDao.java:37)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - com.revature.screens.AddNewClientScreen.start(AddNewClientScreen.java:46)
2018-08-20 19:26:15 ERROR DatabaseDao:43 - com.revature.launcher.Launcher.main(Launcher.java:10)
2018-08-20 19:26:15 WARN  DatabaseDao:45 - failed to create new user
2018-08-20 19:28:08 TRACE DatabaseDao:38 - 1 records created
